extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/WhiteListBasedConverter.java

```diff
@@ -169,15 +170,26 @@ private String getPrefixKey(String key, SortedMap<String, ?> whiteList)
     if (prefixKey == null) {
       return null;
     }
-    ImmutableList.Builder<String> outputList = new ImmutableList.Builder();
+    ImmutableList.Builder<String> outputList = new ImmutableList.Builder<>();
     Set<String> dimensions = whiteListDimsMapper.get(prefixKey);
     if (dimensions == null) {
       return Collections.emptyList();
     }
     for (String dimKey : dimensions) {
-      String dimValue = (String) event.getUserDims().get(dimKey);
-      if (dimValue != null) {
-        outputList.add(GraphiteEmitter.sanitize(dimValue));
+      Object rawValue = event.getUserDims().get(dimKey);
```

Comment:
Same fix also needs to be applied to WhiteListBasedDruidToTimelineEventConverter
@kevinconaway : can you fix that class too ?

Reply:
Sure thing, done.